WireUI/Tests/WireMainNavigationUITests/Coordinator/MainCoordinatorTests.swift

```diff
@@ -87,24 +93,24 @@ final class MainCoordinatorTests: XCTestCase {
 
         // Then
         XCTAssertNil(splitViewController.conversationList)
-        XCTAssertNotNil(tabBarController.conversations?.conversationList)
+        XCTAssertNotNil(tabBarController.conversationList)
     }
 
     @MainActor
-    func testShowingArchivedConversations() {
+    func testShowingArchivedConversations() async {
         // When
-        sut.showArchivedConversations()
+        await sut.showArchive()
 
         // Then
         XCTAssertNil(splitViewController.conversationList)
-        XCTAssertNotNil(tabBarController.conversations?.conversationList)
+        XCTAssertNotNil(tabBarController.conversationList)
         XCTAssertNotNil(splitViewController.archive)
         XCTAssertNil(tabBarController.archive)
     }
 
     // TODO: [WPB-10903] add many more tests, e.g.
```

Comment:
is this done in another PR?

Reply:
I had created a separate ticket to think about it again. Maybe it's not needed.